keras_retinanet/bin/train.py

```diff
@@ -406,6 +431,12 @@ def csv_list(string):
     parser.add_argument('--image-min-side', help='Rescale the image so the smallest side is min_side.', type=int, default=800)
     parser.add_argument('--image-max-side', help='Rescale the image if the largest side is larger than max_side.', type=int, default=1333)
 
+    parser.add_argument('--tensorboxes',  help='Number of images to store on tensorboard', type=int, default=0)
+    parser.add_argument('--use-val-loss', dest='val_loss', help='Compute validation loss', action='store_true')
+    parser.add_argument('--optimizer', help='choose optimizer. default=adam', default='adam', choices=['adam', 'sgd', 'adagrad', 'adadelta', 'adamax', 'nadam', 'rmsprop'])   
```

Comment:
Same here with the `help` section. Also, I would omit the `choose`, simply "The optimizer to use." will be fine.